### ISSUE TITLE
Label only prints first five lines

### ISSUE THREAD

**Issue by MadelineSims** [ExLibrisGroup/SpineOMatic#57]
We have been having this issue where our spine labels will only print the first five lines of a call # even though I have the settings set to 10. We definitely have rooms on the labels. Attached is screen shot. Any suggestions?

Thanks,
Madeline
![Screenshot 2023-02-06 160215](https://user-images.githubusercontent.com/124630856/217085753-67e4ed64-81b2-4b7c-8ee6-6664adb77891.png)

**Comment by mgobat** [comment]
@MadelineSims, could you show me a screenshot of the "Call Number Format" tab? Also, the contents of the "Current XML" tab after you've scanned in the barcode may be helpful. Please scrub out any information from the XML that you would not like public. Thanks.

**Comment by MadelineSims** [comment]
Hello,

Here is XML with some sensitive parts scrubbed. Screen shot as well for tab of call number format. Thanks so much!! Madeline

<?xml version="1.0" ?>
<printout>
<result>
<error> </error> </result>
<form-name>Default</form-name>
<form-language>???</form-language>
<form-format>??</form-format>
<section-01>
<physical_item_display_for_printing>
<mms_id>991003818216606575</mms_id>
<bib_suppress_from_publishing>false</bib_suppress_from_publishing>
<title>Hollow /</title>
<author>Watters, Shannon.</author>
<isbn>1684158524</isbn>
<complete_edition> </complete_edition>
<network_numbers>
<network_number>(OCoLC)1296944701</network_number>
<network_number>(EXLNZ-01FALSC_NETWORK)9914710231906561</network_number> </network_numbers>
<place_of_publication>Los Angeles :</place_of_publication>
<date_of_publication>2022.</date_of_publication>
<publisher_const>Boom! Box a division of Boom Entertainment Inc</publisher_const>
<holding_id>22100920180006575</holding_id>
<holding_suppress_from_publishing>false</holding_suppress_from_publishing>
<calculated_suppress_from_publishing>false</calculated_suppress_from_publishing>
<permanent_call_number_type_name>Library of Congress classification</permanent_call_number_type_name>
<permanent_call_number_type>0</permanent_call_number_type>
<permanent_call_number>PZ7.7.H64 W388 Ho 2022</permanent_call_number>
<call_number_type_name>Library of Congress classification</call_number_type_name>
<call_number_type>0</call_number_type>
<call_number>PZ7.7.H64 W388 Ho 2022</call_number>
<accession_number> </accession_number>
<copy_id> </copy_id>
<in_temp_location>false</in_temp_location>
<temp_call_number_type> </temp_call_number_type>
<temp_call_number> </temp_call_number>
<temp_call_number_source> </temp_call_number_source>
<temp_policy> </temp_policy>
<pid>23100920170006575</pid>
<barcode>33801013285673</barcode>
<creation_date>2023-02-02Z</creation_date>
<modification_date>2023-02-06Z</modification_date>
<base_status_name>Item not in place</base_status_name>
<base_status_code>0</base_status_code>
<awaiting_reshelving>false</awaiting_reshelving>
<physical_material_type_name>Book</physical_material_type_name>
<physical_material_type_code>BOOK</physical_material_type_code>
<policy_name>01 Book Loan</policy_name>
<policy_code>01</policy_code>
<provenance> </provenance>
<po_line>POL-7453</po_line>
<expected_arrival_date>2023-02-02Z</expected_arrival_date>
<year_of_issue> </year_of_issue>
<enum_a> </enum_a>
<enum_b> </enum_b>
<enum_c> </enum_c>
<enum_d> </enum_d>
<enum_e> </enum_e>
<enum_f> </enum_f>
<enum_g> </enum_g>
<enum_h> </enum_h>
<chron_i> </chron_i>
<chron_j> </chron_j>
<chron_k> </chron_k>
<chron_l> </chron_l>
<chron_m> </chron_m>
<break_indicator> </break_indicator>
<pattern_type> </pattern_type>
<linking_number> </linking_number>
<type_of_unit> </type_of_unit>
<issue_level_description> </issue_level_description>
<replacement_cost>15.29</replacement_cost>
<receiving_operator> </receiving_operator>
<process_type_name>Acquisition</process_type_name>
<process_type_code>ACQ</process_type_code>
<inventory_number> </inventory_number>
<inventory_price>15.29</inventory_price>
<alternative_call_number> </alternative_call_number>
<alternative_call_number_type> </alternative_call_number_type>
<storage_location_id> </storage_location_id>
<pages> </pages>
<pieces> </pieces>
<public_note> </public_note>
<fulfillment_note> </fulfillment_note>
<internal_note_1> </internal_note_1>
<internal_note_2> </internal_note_2>
<internal_note_3> </internal_note_3>
<statistics_note_1> </statistics_note_1>
<statistics_note_2> </statistics_note_2>
<statistics_note_3> </statistics_note_3>
<requested>false</requested>
<edition> </edition>
<imprint>Los Angeles : Boom! Box, a division of Boom Entertainment, Inc., 2022.</imprint>
<language>eng</language>
<library_details>
<parsed_call_number>
<call_no_1>PZ</call_no_1>
<call_no_2>7.7</call_no_2>
<call_no_3>.H64</call_no_3>
<call_no_4>W388</call_no_4>
<call_no_5>Ho</call_no_5>
<call_no_6>2022</call_no_6> </parsed_call_number>
<title_abcnph>Hollow / written by Shannon Watters &amp; Branden Boyer-White ; illustrated by Berenice Nelle ; colored by Kaitlyn Musto, Kieran Quigley, Gonçarlo Lopes ; lettered by Jim Campbell.</title_abcnph>
<retention_reason> </retention_reason>
<retention_note> </retention_note>
<current_date>2/7/2023</current_date> </physical_item_display_for_printing> </section-01> </printout>
![Screenshot 2023-02-07 151914](https://user-images.githubusercontent.com/124630856/217356486-ce2feffc-aa62-4d32-8d66-27fb67f08e7c.png)

**Comment by MadelineSims** [comment]
Screen shot of XML with the call # info 
![Screenshot 2023-02-07 152342](https://user-images.githubusercontent.com/124630856/217356822-65afd09b-521e-452a-bd61-9f413a8952ca.png)

**Comment by mgobat** [comment]
@MadelineSims, thanks for the additional details.

Alma is returning `<call_number>PZ7.7.H64 W388 Ho 2022</call_number>` to SpineOMatic. So, it appears that it is dropping the "Ho" portion of the call number when printing the label. 

Here is a screenshot of my test using "Tweak & Test", pasting in the call number returned by Alma. All the call number parts appear on the label. Do you have any options set in the "Tweak & Test" panel that may be causing the problem?

![Screenshot 2023-02-07 153900](https://user-images.githubusercontent.com/45852490/217373720-cf6d6071-6675-44e1-aff9-3efc4b3dc0ee.png)

Thanks.

**Comment by MadelineSims** [comment]
Well, after I checked that my settings were the same as your screenshot, I went to check the printer settings and finally figured out what was going on. The label printer settings were a bit off - they had the size of the labels at 1 in by 1 in. The labels are actually 1 in by 1.5 in and because it was thinking they were smaller, weren't printing out rest of label. So, mystery solved! Thanks for taking the time to help me trouble shoot :)
![Screenshot 2023-02-08 085927](https://user-images.githubusercontent.com/124630856/217550744-1068f9a5-ea5f-460c-9cd4-c182ed1fee6d.png)
